javascript/javascript1/week2/candy.js

```diff
@@ -0,0 +1,48 @@
+// Candy helper 
+const boughtCandyPrices=[];
+function addCandy(candyType, weight){
+const pricePerGram=[0.5, 0.7, 1.1, 0.03];
+    for(let i=0; i<pricePerGram.length; i++){
+        if(candyType=="Sweet"){
+        return boughtCandyPrices.push(weight*pricePerGram[0]);
+        }else if(candyType=="Chocolate"){
+            return boughtCandyPrices.push(weight*pricePerGram[1]);
+        }else if(candyType=="Toffee"){
+            return boughtCandyPrices.push(weight*pricePerGram[2]);
```

Comment:
Interesting approach however I think you could really simplify this function if you use the price per gram amount direction in the argument like 
```js
boughtCandyPrices.push(weight*0.5);
```

which would eliminate the need to loop over anything.